ChangeLog.md

```diff
@@ -4,6 +4,10 @@
 
   *
 
+## v2.5.0 (2022-04-0e) 
```

Comment:
@mushyy let's have the dates in decimal :-)

Reply:
Beep boop